numba/tests/test_array_methods.py

```diff
@@ -196,6 +196,11 @@ def array_dot_chain(a, b):
 def array_ctor(n, dtype):
     return np.ones(n, dtype=dtype)
 
+def array_average(a):
```

Comment:
These tests probably need to go into the `test_np_functions.py` module as `np.average` is not an array method.